src/components/content/common/csp/CspLogo.ts

```diff
@@ -20,12 +20,13 @@ interface CSP {
 }
 
 export const cspMap = new Map<CloudServiceProvider.name, CSP>([
-    [CloudServiceProvider.name.HUAWEI, { name: 'Huawei', logo: HuaWeiLogo }],
-    [CloudServiceProvider.name.AZURE, { name: 'Azure', logo: AzureLogo }],
-    [CloudServiceProvider.name.ALICLOUD, { name: 'Alibaba', logo: AlibabaLogo }],
-    [CloudServiceProvider.name.OPENSTACK, { name: 'Openstack', logo: OpenStackLogo }],
+    [CloudServiceProvider.name.HUAWEI_CLOUD, { name: 'HuaweiCloud', logo: HuaWeiLogo }],
     [CloudServiceProvider.name.FLEXIBLE_ENGINE, { name: 'FlexibleEngine', logo: FlexibleEngineLogo }],
+    [CloudServiceProvider.name.OPENSTACK_TESTLAB, { name: 'OpenstackTestLab', logo: OpenStackLogo }],
+    [CloudServiceProvider.name.REGIO_CLOUD, { name: 'RegioCloud', logo: ScsLogo }],
```

Comment:
we need two different logos for RegioCloud and PlusServer. Both logos are available in the repo already.
![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/b20e007c-f6a9-49c1-86d2-4be55d8e0acb)

Please also rename the Scs_PlusServer_Logo.png to PlusServer_Logo.png